cpp/src/aztec/honk/pcs/gemini/gemini.hpp

```diff
@@ -487,41 +419,20 @@ template <typename Params> class MultilinearReductionScheme {
     /**
      * @brief Computes two commitments to A₀ partially evaluated in r and -r.
      *
-     * @param claims_f array of claims containing commitments to non-shifted polynomials
-     * @param claims_g array of claims containing commitments to shifted polynomials
-     * @param rhos vector of m powers of rho used for linear combination
+     * @param batched_f batched commitment to non-shifted polynomials
+     * @param batched_g batched commitment to to-be-shifted polynomials
      * @param r evaluation point at which we have partially evaluated A₀ at r and -r.
      * @return std::pair<Commitment, Commitment>  c0_r_pos, c0_r_neg
      */
-    static std::pair<Commitment, Commitment> compute_simulated_commitments(
-        std::span<const MLEOpeningClaim<Params>> claims_f,
-        std::span<const MLEOpeningClaim<Params>> claims_g,
-        std::span<const Fr> rhos,
-        Fr r)
+    static std::pair<Commitment, Commitment> compute_simulated_commitments(Commitment& batched_f,
+                                                                           Commitment& batched_g,
+                                                                           Fr r)
```

Comment:
Passing `r_inverse` instead of `r` would save one field inversion.

Reply:
As discussed on slack, there's nothing we can do here